aip/general/0192.md

```diff
@@ -108,6 +108,14 @@ colloquial use that avoiding it would obscure the reference (example: [IBM][]).
 Comments **should** spell and capitalize trademarked names consistent with the
 trademark owner's current branding.
 
+### Deprecations
+
+To deprecate a service or RPC, the `deprecated`
+[option](https://developers.google.com/protocol-buffers/docs/proto#options)
+**must** be set to `true`, and the first line of the respective comment **must**
```

Comment:
Maybe it is not that big a deal, but I would prefer it be the last paragraph rather than the first line. In other words...

```proto
// Frobs the book.
//
// Deprecated: Use FracBook instead.
rpc FrobBook...
```

Thoughts?

Reply:
I would prefer it to be the first thing in the comment, to make it more prominent. It's not a big deal when looking at the proto file or even the online documentation, but for autocomplete in IDEs, the first part of the comment is often much more prominent than the rest.

Reply:
Okay, I can get behind that.

Reply:
+1 to the first item, marking as resolved.